docs/configuring_block_comments.rst

```diff
@@ -122,19 +122,19 @@ To configure the following example...
    rule:
      block_comment_001:
        disable : False
-       header_left : None
+       header_left : ""
```

Comment:
To be consistent, could you change these to single quotes?

Reply:
Done, thanks.